app/Models/Post.php

```diff
@@ -157,4 +155,39 @@ public function toBlinkPayload()
             'deleted_at' => $this->deleted_at ? $this->deleted_at->toIso8601String() : null,
         ];
     }
+
+    /**
+     * Apply the given tag to this post.
+     */
+    public function tag($tagName)
+    {
+        $tag = Tag::where('tag_name', $tagName)->first();
+
+        // Create the tag if it doesn't already exist (so we don't have to create tags separately)
+        if (! $tag) {
```

Comment:
@DFurnes I made that setter but I wasn't sure how it gets called? Right now I have this:
`        $tag = Tag::firstOrCreate(['tag_name' => $tagName], ['tag_slug' => str_slug($tagName, '-')]);`
which looks for the tag with the given name, and if it's not found it creates the tag with the given name and slug. Does it make sense to do it that way?